homeassistant/components/zwave/__init__.py

```diff
@@ -678,12 +682,24 @@ def network_value_changed(self, value):
 
     def value_changed(self, value):
         """Called when a value for this entity's node has changed."""
+        self.update_attributes()
         self.update_properties()
         self.schedule_update_ha_state()
 
+    def update_attributes(self):
```

Comment:
I think this should be protected, i.e. `_update_attributes`

Also, probably call this from constructor instead of setting to None-s